scss/colors.scss

```diff
@@ -1,3 +1,4 @@
 $blue: #00ADEE;
 $green: #BED23C;
 $grey: #C4C4C4;
+$white: #FFFFFF;
```

Comment:
when a color has the format #AABBCC you can write it like #ABC, in this case it should be `#FFF`